### ISSUE TITLE
OOM error while converting a luminance matrix to a RGB grayscale bitmap

### ISSUE THREAD

**Issue by archie94** [guardianproject/haven#274]
Crashlog

```
java.lang.OutOfMemoryError: Failed to allocate a 8294412 byte allocation with 6502296 free bytes and 6MB until OOM
 at org.havenapp.main.sensors.media.ImageCodec.lumaToGreyscale(ImageCodec.java:44)
 at org.havenapp.main.sensors.media.ImageCodec.lumaToBitmapGreyscale(ImageCodec.java:55)
 at org.havenapp.main.sensors.media.MotionAsyncTask.run(MotionAsyncTask.java:102)

```

**Comment by n8fr8** [comment]
Should be fixed in 0.2.0 alpha 2... reduced preview size and improved memory usage overall

**Comment by lukeswitz** [comment]
Seems to be fixed, tested simulator v5.1-P & devices Nexus 4 5.1 & Pixel P. Could not reproduce, memory use down 40% in preview! Excellent

**Comment by jbizzle2** [comment]
jbwolf

**Comment by jbizzle2** [comment]
jbwolf
On Jul 6, 2018 9:17 AM, "Nathan Freitas" <notifications@github.com> wrote:
>
> Should be fixed in 0.2.0 alpha 2... reduced preview size and improved
memory usage overall
